gap/small.gd

```diff
@@ -72,14 +72,48 @@ UnbindGlobal( "SMALL_AVAILABLE" );
 ##  <Func Name="SMALL_AVAILABLE" Arg='order'/>
 ##
 ##  <Description>
-##  returns fail if the library of groups of <A>order</A> is not installed. 
-##  Otherwise a record with some information about the construction of the 
-##  groups of <A>order</A> is returned.
+##  returns fail if the library of groups of order <A>order</A> is not installed.
+##  Otherwise a record with some information about the construction of the
+##  groups of order <A>order</A> is returned.
 ##  </Description>
 ##  </ManSection>
 ##
 DeclareGlobalFunction( "SMALL_AVAILABLE" );
 
+#############################################################################
+##
+#F  SmallGroupsAvailable( <order> )
+##
+##  <#GAPDoc Label="SmallGroupsAvailable">
+##  <ManSection>
+##  <Func Name="SmallGroupsAvailable" Arg='order'/>
+##
+##  <Description>
+##  returns <C>true</C> if the library of groups of order <A>order</A> is
+##  installed, and <C>false</C> otherwise.
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+DeclareGlobalFunction( "SmallGroupsAvailable" );
+
+#############################################################################
+##
+#F  NumberSmallGroupsAvailable( <order> )
+##
+##  <#GAPDoc Label="NumberSmallGroupsAvailable">
+##  <ManSection>
+##  <Func Name="NumberSmallGroupsAvailable" Arg='order'/>
+##
+##  <Description>
+##  returns <C>true</C> if the number of groups of order <A>order</A> is known, and
+##  <C>false</C> otherwise.
```

Comment:
Perhaps make it crystal clear that this affects whether `NumberSmallGroups(order)` will return a value or run into an error? E.g. add "In the former case, the number can be retrieved using `<Ref Func="NumberSmallGroups"/>.`

Also, a short example could illuminate this:
```
gap> NumberSmallGroupsAvailable(100);
gap> NumberSmallGroups(100);
16
gap> NumberSmallGroupsAvailable(2048);
false
gap> NumberSmallGroups(2048);
Error, FOOBAR
```